modules/ui/app/scripts/services/RuleEditFactory.js

```diff
@@ -20,9 +19,7 @@ angular.module('wasabi.services').factory('RuleEditFactory', ['UtilitiesFactory'
                 { menuLabel: 'does not equal', stringForm: '!=', jsonForm: 'notEquals' },
                 { menuLabel: 'equals (same case)', stringForm: '^=', jsonForm: 'stringEqualsExact' },
```

Comment:
Since we annotate this with "same case", it might be an idea to note that equals is case insensitive in the string case? I am not too sure about this, it's fine this way as well (plus has been this way for a long time).